src/code/z_debug.c

```diff
@@ -106,60 +108,59 @@ void Regs_Init(void) {
     }
 }
 
-// Called when free movement is active.
-void func_8006375C(s32 arg0, s32 arg1, const char* text) {
+// Function is stubbed. Name is assumed by similarities in signature to `DbCamera_ScreenTextColored` and usage.
+void DbCamera_ScreenText(u8 x, u8 y, const char* text) {
 }
 
-// Store text during Update, to be drawn later during Draw
-void func_8006376C(u8 x, u8 y, u8 colorIndex, const char* text) {
-    PrintTextBufferEntry* entry;
+void DbCamera_ScreenTextColored(u8 x, u8 y, u8 colorIndex, const char* text) {
+    DbCameraTextBufferEntry* entry = &sDbCameraTextBuffer[sDbCameraTextEntryCount];
     char* textDest;
     s16 charCount;
 
-    entry = &sDebugPrintTextBuffer[sDebugPrintTextBufferNumUsed];
-    if (sDebugPrintTextBufferNumUsed < ARRAY_COUNT(sDebugPrintTextBuffer)) {
+    if (sDbCameraTextEntryCount < ARRAY_COUNT(sDbCameraTextBuffer)) {
         entry->x = x;
         entry->y = y;
         entry->colorIndex = colorIndex;
 
         // Copy text into the entry, truncating if needed
         charCount = 0;
         textDest = entry->text;
+
         while ((*textDest++ = *text++) != '\0') {
             if (charCount++ > (ARRAY_COUNT(entry->text) - 1)) {
                 break;
             }
         }
+
         *textDest = '\0';
 
-        sDebugPrintTextBufferNumUsed++;
+        sDbCameraTextEntryCount++;
     }
 }
 
-// Draw text previously stored by calls to `func_8006376C`
-void func_80063828(GfxPrint* printer) {
+void DbCamera_DrawScreenText(GfxPrint* printer) {
     s32 i;
     Color_RGBA8* color;
-    PrintTextBufferEntry* entry;
+    DbCameraTextBufferEntry* entry;
 
-    for (i = 0; i < sDebugPrintTextBufferNumUsed; i++) {
-        entry = &sDebugPrintTextBuffer[i];
+    for (i = 0; i < sDbCameraTextEntryCount; i++) {
+        entry = &sDbCameraTextBuffer[i];
+        color = &sDbCameraTextColors[entry->colorIndex];
 
-        color = &sDebugPrintTextColors[entry->colorIndex];
         GfxPrint_SetColor(printer, color->r, color->g, color->b, color->a);
         GfxPrint_SetPos(printer, entry->x, entry->y);
         GfxPrint_Printf(printer, "%s", entry->text);
     }
 }
 
-// Process inputs to control the reg editor
 void Regs_UpdateEditor(Input* input) {
```

Comment:
idk I like that comment sadge
You could make it a `/** * */` doc comment though